zazu/util.py

```diff
@@ -95,20 +95,30 @@ def prompt(text, default=None, expected_type=str):
     return expected_type(result)
 
 
-def pick(choices, message):
+def pick(choices, message, checkbox=False):
+
     if len(choices) > 1:
         click.clear()
-        questions = [
-            inquirer.List(' ',
-                          message=message,
-                          choices=choices,
-                          ),
-        ]
-        response = inquirer.prompt(questions)
-        if response is None:
-            raise KeyboardInterrupt
-        return response[' ']
-    return choices[0]
+        if not checkbox:
```

Comment:
To save some code you could do:
    
    inquirer_type = inquirer.Checkbox if checkbox else inquirer.List
    questions = [inquirer_type(' ', message = message, choices = choices)]